src/messages/AdaptiveCards/styles.module.css

```diff
@@ -0,0 +1,189 @@
+.ac-pushbutton,
+.ac-pushbutton.subdued {
+	background-color: transparent;
+	border: 1px solid primaryColor;
+	color: primaryColor;
+
+	cursor: pointer;
+	height: 40px;
+
+	padding: 10px 20px;
+	border-radius: 14px;
+}
+
+.ac-pushbutton.style-default.expandable.expanded {
+	background: primaryGradient;
+	color: primaryContrastColor;
+	border: 1px solid primaryColor;
+
+	cursor: pointer;
+	height: 40px;
+
+	padding: 10px 20px;
+	border-radius: 14px;
+}
+
+/*.ac-input {
+  border: 1px solid ${theme.primaryColor};
+  height: 40;
+  padding: ${theme.unitSize}px ${theme.unitSize * 2}px;
+}*/
+
+.adaptivecard-wrapper {
+	width: 100%;
+	box-sizing: border-box;
+	max-width: 100%;
+	padding: 16px !important;
+}
+
+.adaptivecard-wrapper > *:focus {
+	outline: 1px solid var(--primary-color);
+	border-color: var(--primary-color);
+}
+
+.adaptivecard-wrapper button {
```

Comment:
I think we are also missing the button styles when is disabled

Reply:
fixed